_includes/pbc/all/install-features/202410.0/install-the-amazon-quicksight-module.md

```diff
@@ -0,0 +1,596 @@
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+- Verify `QuicksightAnalyticsCollectionExpanderPlugin`: go to `http://mysprykershop.com/analytics-gui/analytics` and make sure the page is expanded by Quicksight analytics, by default you should see the `No Analytics permission has been granted to the current user` message.
+- Verify `QuicksightUserExpanderPlugin`: create a new Quicksight user in the `spy_quicksight_user` DB table, call `UserFacade::getUserCollection()` for a user used for newly created Quicksight user and make sure the `UserCollection.user.quicksightUser` is expanded.
+- Verify `DeleteQuicksightUserPostUpdatePlugin`: log in to Back Office, go to the **Users** -> **Users** section, deactivate a user used for newly created Quicksight user and make sure the corresponding row is deleted in the `spy_quicksight_user` DB table.
+
+{% endinfo_block %}
+
+2. Enable behaviors by registering the console commands:
+
+| PLUGIN                          | SPECIFICATION                                                                                            | PREREQUISITES | NAMESPACE                                             |
+|---------------------------------|----------------------------------------------------------------------------------------------------------|---------------|-------------------------------------------------------|
+| QuicksightUserSyncCreateConsole | Persists in `spy_quicksight_user` DB table users registered in quicksight by persisted user emails.      |               | SprykerEco\Zed\AmazonQuicksight\Communication\Console |
+
+**src/Pyz/Zed/Console/ConsoleDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\Console;
+
+use Spryker\Zed\Console\ConsoleDependencyProvider as SprykerConsoleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
+use SprykerEco\Zed\AmazonQuicksight\Communication\Console\QuicksightUserSyncCreateConsole;
+
+class ConsoleDependencyProvider extends SprykerConsoleDependencyProvider
+{
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return list<\Symfony\Component\Console\Command\Command>
+     */
+    protected function getConsoleCommands(Container $container): array
+    {
+        return [
+            new QuicksightUserSyncCreateConsole(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+To check the `quicksight-user:sync:create` console command:
+
+1. Create new Quicksight user in Quicksight.
+2. Run the following command:
+```bash
+console quicksight-user:sync:create
+```
+3. In the `spy_quicksight_user` table check that the corresponding Quicksight user has been added.
+
+{% endinfo_block %}
+
+{% info_block infoBox "Deployment" %}
+
+In order users to be synchronized automatically during deployment, you can configure the installation stage of the pipeline, for example the `destructive` pipeline. Add the following command to the end of the `demodata` section:
+
+**config/install/destructive.yml**
+```json
```

Comment:
yaml

Reply:
we are providing .yml in the demoshops by default so I would leave as it

Reply:
fixed